frontend2/.env.development

```diff
@@ -1,3 +1,3 @@
 REACT_APP_THIS_URL=http://localhost:3000
-REACT_APP_BACKEND_URL=http://api.staging.battlecode.org/
+REACT_APP_BACKEND_URL=http://localhost:8000
```

Comment:
i accidentally changed this to staging in a recent pr, it's good that we're changing it back to localhost here